go.mod

```diff
@@ -1,21 +1,21 @@
 module github.com/testcontainers/moby-ryuk
 
-go 1.21
+go 1.23
 
 require (
-	github.com/docker/docker v27.1.1+incompatible
+	github.com/docker/docker v27.2.0+incompatible
```

Comment:
I'm fine with this bump, although there are some conversations with other maintainers regarding the default docker version, as it could limit the features Ryuk could provide regarding the docker versions in the workers in which the different tc-libs run.

Not a blocker though. 

Cc/ @kiview

Reply:
Also as a general rule I find updating regularly causes less pain and keeps on top of any security issues.

I believe the main docker release is now following semantic versioning and moving towards module support (can't come soon enough), but I'm sure you know more than I do on that 😄